Scripts/LevelManager.gd

```diff
@@ -92,6 +94,10 @@ func show_offline_screen():
 	var gos = Offline_screen.instance()
 	get_parent().call_deferred("add_child", gos)
 
+func show_silver_moon_label():
+	var gos = Silver_moon_label.instance()
```

Comment:
Name gos is acronim of GameOverScreen, so it has no sens in this case. Maybe `sm_label` (SilverMoon Label) will be better?

Reply:
Yes